eng/pipelines/common/platform-matrix.yml

```diff
@@ -27,7 +27,6 @@ parameters:
 jobs:
 
 # Linux arm
-
```

Comment:
NIT: I assume this was unintentional?

Reply:
Sigh, happens every time, in my past PR as well. Will leave as is, as it is just a blank line to not reset CI and it seems reasonable to have the comment together with the section it refers to.